tools/wwwspace/prune.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+#
+# Node.js www server space pruning script.
+# Runs against a subdirectory of /home/dist/nodejs and removes old builds:
+# - For anything over 2 calendar years ago, retain only the build dated first of the month
+# - For anything in last 2 calendar years but not the last two months, retain date numbers ending in 1
+# - Keep everything from the last two months.
+#
+
+[ $# -lt 2 ] && echo Usage: $0 '[nightly|v8-canary]' '[show|delete]' && exit 1
+PRODUCT=$1
+[ "$1" != "nightly" -a "$1" != "v8-canary" ] && echo Parameter mist be one of: nightly v8-canary && exit 1
```

Comment:
```suggestion
[ "$1" != "nightly" -a "$1" != "v8-canary" ] && echo Parameter mist be one of: nightly v8-canary && exit 1
PRODUCT=$1
```

not important, just aesthetic